src/SimpleThings/EntityAudit/RevisionCriteria.php

```diff
@@ -0,0 +1,9 @@
+<?php
+namespace SimpleThings\EntityAudit;
+
+class RevisionCriteria
+{
+    protected $dateFrom;
+    protected $dateTo;
+    protected $username;
```

Comment:
OK so I left RevisionCriteria properties as protected and add getters and setters to force the type when necessary, is that ok for you?